app/main.py

```diff
@@ -1,34 +1,162 @@
+from app.custom_exceptions import (
+    LocationError,
+    NumberOfTypesError,
+    NumberOfShipsError
+)
+from app.standard import (
+    art_deck,
+    art_field,
+    border_d,
+    border_l,
+    border_r,
+    border_u,
+    hit,
+    miss,
+    sunk,
+
+
+)
```

Comment:
Maybe better like this 
```suggestion
from app.standard import *
```

Reply:
flake8 does not allow